Makefile

```
@@ -230,7 +230,7 @@ RUNS ?= 1
 EVAL_CONFIG ?= eval.yaml
 
 .PHONY: run-mcpchecker-eval
-run-mcpchecker-eval: $(MCPCHECKER) ## Run mcpchecker eval (TASK=name, CATEGORY=..., EVAL_CONFIG=eval-logs.yaml, RUNS=3)
+run-mcpchecker-eval: $(MCPCHECKER) ## Run mcpchecker eval (TASK=name, CATEGORY=..., EVAL_CONFIG=evals/mcpchecker/eval.yaml, RUNS=3)
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)